src/index.html

```diff
@@ -13,6 +13,57 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--yellow">
+
+      <div class="stars__row">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+
+      <div class="stars__row">
```

Comment:
Add empty lines between multiline sibling blocks of HTML. There should be an empty line before this div to separate it from the previous sibling div.